.github/workflows/backport.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  pull_request_target:
+    branches:
+      - master
+    types:
+      - labeled
+      - closed
+
+jobs:
+  backport:
+    name: Backport PR
+    if: |
+      github.event.pull_request.merged == true
+      && contains(github.event.pull_request.labels.*.name, 'auto-backport')
+      && (
+        (github.event.action == 'labeled' && github.event.label.name == 'auto-backport')
+        || (github.event.action == 'closed')
+      )
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: 'elastic/kibana-github-actions'
+          ref: main
+          path: ./actions
+
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+
+      - name: Run Backport
+        uses: ./actions/backport
+        with:
+          github_token: ${{secrets.ELASTICSEARCHMACHINE_TOKEN}}
+          commit_user: elasticsearchmachine
+          commit_email: elasticsarchmachine@users.noreply.github.com
+          auto_merge: 'false'
```

Comment:
what does this _auto_merge_ mean here? Does it match what we do now have with the `auto-merge` label? I'd expect this backport PR contains the auto-merge label too

Reply:
In this context "auto merge" is the [GitHub feature](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request). We won't be leveraging that since it depends on status checks being _required_, which we don't do. Instead we'll use the `auto-merge` label.

To start with, we won't put the auto-merge label on backports by default but we might in the future.